packages/qmongjs/src/helpers/functions/defineLevel.ts

```diff
@@ -28,3 +27,32 @@ export const level = (indicatorData: Indicator | DataPoint) => {
     return "M";
   }
 };
+
+export const level2 = (indicatorData: IndicatorData, dataPoint: DataPoint) => {
+  const { levelGreen, levelYellow, levelDirection } = indicatorData;
+  if (
+    levelGreen === undefined ||
+    levelGreen === null ||
+    levelDirection === null ||
+    dataPoint.var == null
+  ) {
+    return;
+  }
+  const value = dataPoint.var;
+  // If levelYellow is NULL: set to levelGreen
+  const yellow = levelYellow != null ? levelYellow : levelGreen;
+
+  if (
+    (levelDirection === 0 && value <= levelGreen) ||
+    (levelDirection != 0 && value >= levelGreen)
+  ) {
+    return "H";
+  } else if (
+    (levelDirection === 0 && value > yellow) ||
+    (levelDirection != 0 && value < yellow)
+  ) {
+    return "L";
+  } else {
+    return "M";
+  }
+};
```

Comment:
Ble litt vel mye kopi av `level`-funksjonen:) Vil det være mulig å bruke `level` med litt omskriving?

Reply:
Fant ingen umiddelbar løsning på det. Funksjonen vet ikke om det kommer inn et objekt med `levelGreen` eller `level_green`. Logikken kunne derimot vært flyttet til en egen funksjon som gjenbrukes av begge `level`-funksjonene.

Reply:
Noe sånt: https://github.com/mong/mongts/blob/65b39dd9d867b2dc46762562ea2c8a258a29c58e/packages/qmongjs/src/helpers/functions/defineLevel.ts#L3-L51

Reply:
Ja:) Men kan du ikke bruke `getLevel` direkte, i stedet for `level2`? Og bare slette `level2`?

Reply:
Sånn? 

![image](https://github.com/mong/mongts/assets/64078729/7a9520c8-514b-43d7-970a-bb6d2997a611)